kolibri/plugins/coach/assets/src/views/recent-page/channel-list.vue

```diff
@@ -28,10 +32,44 @@
 
   module.exports = {
     name: 'channelList',
-    $trNameSpace: 'channelList',
+    $trNameSpace: 'coachRecentPageChannelList',
     $trs: {
       channels: 'Channels',
       channelList: 'Channel list',
+      lastActive: 'Last active',
+      daysPassed: 'X day(s) ago',
+      monthsPassed: 'X month(s) ago',
+      timePassed:
```

Comment:
$tr values cannot be expressions. Write on the same line and add the eslint ignore line length flag